waffle/templatetags/waffle_tags.py

```diff
@@ -41,16 +41,15 @@ def render(self, context):
     def handle_token(cls, parser, token, kind, condition):
         bits = token.split_contents()
         if len(bits) < 2:
-            raise template.TemplateSyntaxError("%r tag requires an argument" %
-                                               bits[0])
+            raise template.TemplateSyntaxError(f"{bits[0]!r} tag requires an argument")
```

Comment:
suggestion: use `repr(bits[0])` for readability in the spirit of https://peps.python.org/pep-0498/#s-r-and-a-are-redundant

Reply:
This PR is about automatic changes, not manual changes.  Turning on different rules may make other automatic changes.